core/bin/contract-verifier/src/main.rs

```diff
@@ -25,6 +25,32 @@ struct Opt {
     secrets_path: Option<PathBuf>,
 }
 
+async fn perform_storage_migration(pool: &ConnectionPool<Core>) -> anyhow::Result<()> {
```

Comment:
OK, this is very slow (6sec per 1000 rows on a high-end CPU). Working on fixing the migration speed.

Reply:
Reworked the migration logic:
- Got rid of `OFFSET` in select query
- Got rid of `serde_json::Value` roundtrip
- Added rayon for parallel calculation of identifiers
- Replaced `INSERT` with `COPY FROM STDIN`

All of that combined reduced the migration time on mainnet data from ~19 minutes to ~3.5 minutes.

Reply:
I completely agree with `rayon` being completely unnecessary, but it's already a transitive dependency through `zksync_l1_contract_interface` (together with crates such as `circuit_definitions`, `fflonk`, `zksync_kzg`, and many others that also feel unnecessary), so we're not making anything worse here `¯\_(ツ)_/¯`

This being blocked is not an issue at all, because the migration is blocking and nothing will be executed in parallel until it's complete.